### PR TITLE
initial commit with 3 projects

### PR DESCRIPTION
**Pull request by dndhinoja**

### CODE REVIEW

File: ExaminationModelEntity/pom.xml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.11</version>
+      <scope>test</scope>
+    </dependency>
+    <!-- https://mvnrepository.com/artifact/org.hibernate.javax.persistence/hibernate-jpa-2.0-api -->
```
Comment (nddhinoja) [inline ExaminationModelEntity/pom.xml:28]: Remove additional comments.

**Review by nddhinoja — CHANGES_REQUESTED** [review]
Please address review comments

File: ExaminationModelEntity/pom.xml (hunk 1 above)
Comment (dndhinoja) [reply ExaminationModelEntity/pom.xml:28]: Done

**Review by nddhinoja — APPROVED** [review]
Initial approval.

**Pull request merged**